src/main/java/ca/intelliware/ihtsdo/mlds/repository/CustomAuditEventRepository.java

```diff
@@ -1,71 +1,71 @@
 package ca.intelliware.ihtsdo.mlds.repository;
-
-import java.util.*;
-
-import javax.inject.Inject;
-
-import org.apache.commons.lang.NotImplementedException;
-import org.joda.time.Instant;
-import org.springframework.boot.actuate.audit.AuditEvent;
-import org.springframework.boot.actuate.audit.AuditEventRepository;
-import org.springframework.context.annotation.Bean;
-import org.springframework.stereotype.Repository;
-
-import ca.intelliware.ihtsdo.mlds.config.audit.AuditEventConverter;
-import ca.intelliware.ihtsdo.mlds.domain.PersistentAuditEvent;
-
-/**
- * Wraps an implementation of Spring Boot's AuditEventRepository.
- */
-@Repository
-public class CustomAuditEventRepository {
```

Comment:
@jaykamal are audit logs still written or has this functionality been removed completely?

Reply:
@dmcgihtsdo While doing SB3 Migration, We were trying random solution to address the issues encountered. so we have commented the code forgot to uncomment it. now its uncommented and verified the activity logs. it works fine. Thanks.